compiler/lineinfos.nim

```diff
@@ -195,6 +197,9 @@ const
   warnMax* = pred(hintSuccess)
   hintMin* = hintSuccess
   hintMax* = high(TMsgKind)
+  rstWarnings* = {warnRedefinitionOfLabel, warnUnknownSubstitutionX,
```

Comment:
optional, and could be in future work: could all these be renamed with warnRst prefix for clarity?
this shouldn't impact users

Reply:
may be, but separate warnings allow to turn each one to an error (or vice versa suppress it). And it's not that hard to maintain the list.

Reply:
no, i meant renaming them eg:
warnRedefinitionOfLabel => warnRstRedefinitionOfLabel
warnUnknownSubstitutionX => warnRstUnknownSubstitutionX
etc

Reply:
makes sense, agree. Done for errors and warnings.